menu_gold.rb

```diff
@@ -0,0 +1,66 @@
+#setting up arrays for each of the variables
+ingredient = ["baby banana", "Runts banana", "corn ball", "plantain", "classic Dole banana", "manzano banana", "burro banana", "goldfinger banana", "ladyfinger banana", "golden beauty banana"]
+prepared = ["seared", "fried", "braised", "corn-balled", "dipped", "grilled", "steamed", "sauteed", "smoked", "baked"]
+adjective = ["chocolatey", "cheesy", "candied", "salted", "spicy", "sugar-coated", "firey", "crunchy", "caramelized", "breaded"]
```

Comment:
I'm impressed that you listed all your array elements consistently (comma followed by space, all double quotes, no dangling elemtns). As someone with borderline ocd that reviews code all day, I appreciate it :tada: